shell/components/nav/Group.vue

```diff
@@ -425,9 +419,13 @@ export default {
     A:focus {
       outline: none;
     }
+
+    &.root {
+      background: transparent;
+      A {
+        padding-left: 14px;
+      }
+    }
   }
 
-  // .bg-yellow {
```

Comment:
This PR CI is passing and I'd really like to get it in, so will approve. The next PR you make though, can you sneak in the removal of this comment?

Reply:
I will do that